torch_np/_ndarray.py

```diff
@@ -29,24 +38,58 @@ def ndim(self):
 
     @property
     def dtype(self):
-        return self._tensor.dtype
+        return _dtypes.dtype_from_torch(self._tensor.dtype)
 
     @property
     def strides(self):
         return self._tensor.stride()   # XXX: byte strides
 
+    @property
+    def base(self):
+        return self._base
```

Comment:
This might create mighty gc cycles,not sure if this is a problem. Or how to avoid it if it is.

Reply:
Why would it create gc cycles? I think that the ownership structure here is clear.